systest/ldbc/ldbc_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+type TestCases struct {
+	Tag   string `yaml:"tag"`
+	Query string `yaml:"query"`
+	Resp  string `yaml:"resp"`
+}
+
+func TestQueries(t *testing.T) {
+	dg, err := testutil.DgraphClient(testutil.ContainerAddr("alpha1", 9080))
+	if err != nil {
+		t.Fatalf("Error while getting a dgraph client: %v", err)
+	}
+
+	yfile, _ := ioutil.ReadFile("test_cases.yaml")
+
+	tc := make(map[string]TestCases)
+
+	err = yaml.Unmarshal(yfile, &tc)
+
+	if err != nil {
+		t.Fatalf("Error while greading test cases yaml: %v", err)
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	for _, tt := range tc {
+		desc := tt.Tag
+		// TODO(anurag): IC06 and IC10 have non-deterministic results because of dataset.
+		// Find a way to modify the queries to include them in the tests
+		if desc == "IC06" || desc == "IC10" {
```

Comment:
Query `IC06` and `IC10` can have multiple correct answers based on the indexing. Will have to modify the queries to get deterministic results. Added a `TODO`.